packages/repeater/src/lib/RepeaterFactory.ts

```diff
@@ -86,11 +80,16 @@ export class RepeaterFactory {
       );
     }
 
-    const bus = container.resolve<EventBus>(EventBus);
-
-    await bus.init?.();
-
-    return bus;
+    container.register<DefaultRepeaterServerOptions>(
+      DefaultRepeaterServerOptions,
+      {
+        useValue: {
+          uri: `${this.configuration.api}/workstations`,
+          token: this.configuration.credentials?.token as string,
```

Comment:
There is assertion above

```suggestion
          token: this.configuration.credentials.token,
```